src/PerfectGym.AutomergeBot.Tests/SlackNotificationsTests.cs

```diff
@@ -72,7 +87,7 @@ public void MessageProvider_ForOpenPR_DurationIsCalculated(int days,int hours,in
         }
 
         [Test]
-        public void MessageProvider_ForMultipleOpenPR_ResolveConflictPhraseOccursOnce()
+        public void MessageProvider_ForTwoOpenPR_ResolveConflictPhraseOccursOnceAndPROpenOccursTwice()
```

Comment:
```suggestion
        public void MessageProvider_ForTwoOpenPR_ResolveConflictPhraseOccursOnceAndPROpenPhraseOccursTwice()
```

Według ksiązkowego podejścia to powinny być 2 oddzielne testy jednostkowe, ale myślę że może zostać tak, jak jest.

Reply:
To jako test zawartości message'a z notyfikacją dla  i dlatego zostawiłem 2 asercje w jednym teście.
Zmienię nazwę testu, tak żeby zawierała warunek dla jakiego to jest wykonywane, może wtedy będzie czytelniej
`MessageProvider_WhenUserHasTwoOpenPR_ResolveConflictPhraseOccursOnceAndPROpenOccursTwice()`